designer/server/src/test/scala/pl/touk/nussknacker/ui/validation/ProcessValidationSpec.scala

```diff
@@ -518,16 +522,52 @@ class ProcessValidationSpec extends AnyFunSuite with Matchers {
     )
   }
 
+  test("handles validation when fragment params are invalid") {
+    val subprocess = CanonicalProcess(
+      MetaData("sub1", FragmentSpecificData()),
+      nodes = List(
+        FlatNode(SubprocessInputDefinition("in", List(SubprocessParameter("subParam1", SubprocessClazzRef[String])))),
+        FlatNode(SubprocessOutputDefinition("subOut1", "out", List(Field("foo", "42L"))))
+      ),
+      additionalBranches = List.empty
+    )
+
+    val process = createProcess(
+      nodes = List(
+//        Source("source", SourceRef(sourceTypeName, List())),
+        Source("source", SourceRef("sourceWithParams", List(evaluatedparam.Parameter("mandatory", "")))),
+        SubprocessInput("subIn", SubprocessRef(
+          subprocess.id,
+          List(evaluatedparam.Parameter("subParam1", ""))),
```

Comment:
Please fix this test to check if mandatory validator is executed and detects null/empty param value (maybe change parameter name to "shouldNotBeEmptyParam").